### DISCUSSION TITLE
lifestyle/programmer-lie-down/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#57] [Announcements]
# lifestyle/programmer-lie-down/

在“内卷”浪潮的冲涮下，看到有不少年轻人选择躺平来应对压力，其实“内卷”和“躺平”这两个现象是一直存在的，只是最近随着互联网信息的发展，人们才注意到这些，并冠以这两个生动的名词。程序员是最先体会到内卷的群体，因为自始至终，程序员都需要保持学习的能力，才能在内卷中不被淘汰，如果感受到沉重的压力，那么躺平是正确的选择。

https://zxuqian.cn/lifestyle/programmer-lie-down/

**Comment by xiaowangxiaowang256256** [comment]
躺平不等于自暴自弃，而是给自己健康和技能充电，积极的改变你的一生。🧡
